src/main/java/org/broadinstitute/consent/http/service/DaaService.java

```diff
@@ -162,13 +162,13 @@ public void sendDaaRequestEmails(User user, Integer daaId) throws Exception {
       if (signingOfficials.isEmpty()) {
         throw new NotFoundException("No signing officials found for user: " + user.getDisplayName());
       }
-      int userId = user.getUserId();
-      String userName = user.getDisplayName();
       for (SimplifiedUser signingOfficial : signingOfficials) {
         DataAccessAgreement daa = findById(daaId);
         String daaName = daa.getFile().getFileName();
-        emailService.sendDaaRequestMessage( signingOfficial.displayName, signingOfficial.email,
-            userName, daaName, daaId, userId);
+        User toUser = new User();
+        toUser.setEmail(signingOfficial.email);
+        toUser.setDisplayName(signingOfficial.displayName);
```

Comment:
This change was made so that downstream code can use `User`; it might be better to change the email code to support `SimplifiedUser` instead